src/rules/rename-default-import.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @fileoverview Rule to enforce aliases for default imports
+ * @author Michał Kołodziejski
+ */
+
+import docsUrl from '../docsUrl'
+
+
+function isDefaultImport(specifier) {
+  if (specifier.type === 'ImportDefaultSpecifier') {
+    return true
+  } else if (specifier.type === 'ImportSpecifier' && specifier.imported.name === 'default') {
+    return true
+  }
+  return false
+}
+
+function isCommonJSImport(declaration) {
+  const variableInit = declaration.init
+  if (variableInit.type === 'CallExpression') {
+    return variableInit.callee.name === 'require'
```

Comment:
we should also ensure that the `require` is a global one, and not `function (require) { require(); }`.

Reply:
this doesn't seem addressed yet?

Reply:
I did in the line 136 (didn't want to pass the context so I did it there).